src/shared/components.tsx

```diff
@@ -0,0 +1,46 @@
+import { classes } from "@/shared/tw";
+import {
+  InputHTMLAttributes,
+  LabelHTMLAttributes,
+  SelectHTMLAttributes,
+  FormHTMLAttributes,
+  HTMLAttributes,
+} from "react";
+
+const focusStyles = "focus:ring-2 focus:ring-inset focus:ring-indigo-600";
+
+export const Title = (props: HTMLAttributes<HTMLHeadingElement>) => {
+  const styles = ` text-base text-center font-semibold leading-7 text-white
```

Comment:
```suggestion
  const styles = `text-base text-center font-semibold leading-7 text-white
```

just a dangling empty space